app/docs/md/learn/concepts/styling/modular-scales.md

```diff
@@ -0,0 +1,144 @@
+
+Fluid scales allow font sizes, margins, padding, and other aspects of layouts to scale in size gradually across a range of viewport sizes, as opposed to changing suddenly at discrete breakpoints via media queries. Taking the previous example with our `h1`, a fluid scale would allow the heading’s font size to scale gradually between 2rem and 4rem based on the current viewport size (as long as that viewport size is between your chosen minimum and maximum width). This approach has been popularized by tools like [Utopia](https://utopia.fyi/).
+
+In addition to gracefully scaling across a range of viewport widths, using fluid type and spacing can reduce the amount of CSS you need to write (especially when it comes to CSS scoped with media queries). Of course, this strategy requires alignment between designers and developers; [the Utopia blog](https://utopia.fyi/blog) has some great reading on this subject.
+
+## `typeScale` and `fontScale`
+
+Enhance Styles uses fluid scales for its font size utility classes via our `typeScale` configuration, and for margins, padding, and gaps (for use in flexbox and grid layouts) via our `spaceScale`. We also emit custom properties for each step of the type and space scales, so that these values can be used in your own [element styles](/docs/learn/concepts/styling/element-styles).
+
+These fluid scales are [customizable](/docs/learn/concepts/styling/customization) via the configuration file. However, [the default configuration](#defaults) comes with some sensible defaults baked in. The default values for each of these two scales are identical; however, the ability to specify unique values for each is offered in order to accommodate a wide range of preferences.
+
+When reviewing or customizing the `typeScale` and `spaceScale`, the key concepts to be aware of are:
+
+### Steps
+
+Type and space scales contain a configurable number of steps. This number should be large enough to provide the designer and developer with a sufficient range of options for setting type size and layout spacing, but not so large that an excessive number of unused steps are generated (as this will bloat the CSS bundle and cause confusion for implementers).
+
+<doc-callout level="info">
+
+For **type scales**, the `steps` option indicates the base step plus the number of positive steps to be generated. For example, 6 steps would produce 1 base step plus 5 positive steps. **Two negative steps will also be generated**, for setting type at sub-body copy sizes.
+
```

Comment:
Please call out **why** only two negative steps are generated. As the type becomes illegible at too small a size.